### ISSUE TITLE
False positive "differs only by captalization" on alignment fields in struct

### ISSUE THREAD

**Issue by misha-drozd** [mgechev/revive#1535]
**Describe the bug**

A struct with more than one `_` field (e.g. for alignment) is reported to have fields differing only by capitalization.

**To Reproduce**

```shell
% cat a.go 
// package main is a package
package main

type siginfoChild struct {
	signo    int32
	errno    int32
	exitCode int32
	_        int32

	pid    int32
	uid    uint32
	status int32

	_ [25]uint32 // Pad to 128 bytes
}

% go run github.com/mgechev/revive@HEAD -config config.toml a.go
a.go:14:2: Field '_' differs only by capitalization to other field in the struct type siginfoChild
```

```toml
enableAllRules = true
```

**Expected behavior**

A struct may have more than one `_` field.

**Desktop (please complete the following information):**

`go version go1.25.1 darwin/arm64`
